README.md

```diff
@@ -40,6 +40,23 @@ Optionally you can pass a list of services to be rebuilt:
 ./bootstrap.sh up api web
 ```
 
+## Services pre-run setup
+Some services will need to perform a certain set of actions before they are able to start successfully. This might include eg. running migrations, setting up fixtures, etc.
+
+To accomodate this, this script will run all `./setup/*.sh` scripts defined, every time they change ssince the last setup run (as per last git modification timestamp).
+
+Setup step will run right after `up` command.
+
+Setup can be forced (regardless of the timestamps) by running:
+
+```bash
+./bootstrap.sh setup
+```
+
+*NOTE*: setup scripts *must* be idempotent (able to run multiple times without adverse effects), as they will run multiple times - on each change to the setup scripts.
+
+*NOTE 2*: each setup script will be run up to 10 times, with a delay of 3 seconds (determined by it's exit code).
+
```

Comment:
Note #1 It's quite an important, IMO. Maybe instead of saying it in a note, we can add it to the description somewhere at the beginning?